client/templates/articles.coffee

```diff
@@ -11,13 +11,46 @@ Template.articles.helpers
   diseases: ->
     Template.instance().diseases.get()
 
+  articlesReady: ->
+    Template.instance().articlesReady.get()
+
 Template.articles.events
-  "change #diseases": (event) ->
+
+  "click .block": (event, template) ->
+    $(".block").removeClass("current")
```

Comment:
As long as the .block element is within the `articles` template, it's better to use `template.$(...` here.